patches/net-cookies-cookie_monster.cc.patch

```diff
@@ -0,0 +1,52 @@
+diff --git a/net/cookies/cookie_monster.cc b/net/cookies/cookie_monster.cc
+index 140b61a81dcd748aa22e9dec472f1d375bebcfae..527470acda3863843bca0bcb93e918aa2c35961a 100644
+--- a/net/cookies/cookie_monster.cc
++++ b/net/cookies/cookie_monster.cc
+@@ -579,6 +579,7 @@ void CookieMonster::AttachAccessSemanticsListForCookieList(
```

Comment:
I think all of these can be eliminated by subclassing CookieMonster and doing a chromium_src override in content::CreateCookieStore

Reply:
I did consider subclassing `CookieMonster` and maybe that does make sense. The issue is that `content::CreateCookieStore` is not always used to create the `CookieStore`. Crucially, it isn't used for navigation cookies and instead the constructor is called directly. See `net/url_request/url_request_context_builder.cc` and in `services/network/network_context.cc` (using `make_unique`). If we simply override the factory, there is a chance that this will require more work rebasing onto new versions of Chromium.

Reply:
I chatted on slack with @bridiver about this and he clarified that by using chromium_src override, we should be able to create a subclass called `CookieMonster` and then use macros to rename the original CookieMonster. I will give this a shot.